<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1407</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter_value_row">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>&lt;h4&gt;&lt;font color='DodgerBlue'&gt; SVC Combo / Date &lt;/font&gt;&lt;/h4&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tw_variableObjListSvcCombo">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Variable ID</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Variable Desc</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="topMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;h4&gt;&lt;font color='DodgerBlue'&gt; DataList &lt;/font&gt;&lt;/h4&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tw_variableObjList">
             <property name="editTriggers">
              <set>QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Variable ID</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Variable Desc</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="foreground">
               <brush brushstyle="NoBrush">
                <color alpha="255">
                 <red>255</red>
                 <green>99</green>
                 <blue>71</blue>
                </color>
               </brush>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="sw_variableObjList">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&lt;h4&gt;&lt;font color='DodgerBlue'&gt; Variable Row Index &lt;/font&gt;&lt;/h4&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QTableWidget" name="tw_variableObjRowList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Chk</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Target</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Description</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Data List ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Row</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Step ID</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QGroupBox" name="grp_variableRowIndex">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>70</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_15">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Row Index</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QFrame" name="frame_rowIndex">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>70</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout_14">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QStackedWidget" name="sw_rowIndex">
                   <property name="currentIndex">
                    <number>0</number>
                   </property>
                   <widget class="QWidget" name="page_4">
                    <layout class="QGridLayout" name="gridLayout_15">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QFrame" name="frame_rowIndexOption1">
                       <layout class="QGridLayout" name="gridLayout_16">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item row="0" column="0">
                         <widget class="QSpinBox" name="sb_rowIndex"/>
                        </item>
                        <item row="0" column="1">
                         <spacer name="horizontalSpacer_2">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                   <widget class="QWidget" name="page_8">
                    <layout class="QGridLayout" name="gridLayout_17">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="0" column="0">
                      <widget class="QFrame" name="frame_rowIndexOption2">
                       <layout class="QGridLayout" name="gridLayout_18">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>0</number>
                        </property>
                        <item row="0" column="0">
                         <widget class="QComboBox" name="cb_rowIndexColumnId"/>
                        </item>
                        <item row="0" column="1">
                         <widget class="QLineEdit" name="edt_rowIndexColumnValue"/>
                        </item>
                       </layout>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QComboBox" name="cb_rowMethod">
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>0</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>Fix</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>By Value</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="variable_layout"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btn_variableRowIndexSave">
     <property name="text">
      <string>Variable Row Index Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_removeVariable">
   <property name="text">
    <string>Remove Variable</string>
   </property>
   <property name="toolTip">
    <string>Remove Variable</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_rowMethod</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>sw_rowIndex</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>1064</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_rowMethod</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>btn_variableRowIndexSave</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sb_rowIndex</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>btn_variableRowIndexSave</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_rowIndexColumnId</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>btn_variableRowIndexSave</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_rowIndexColumnValue</sender>
   <signal>textChanged(QString)</signal>
   <receiver>btn_variableRowIndexSave</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>921</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
